src/sparseml/transformers/finetune/data/custom.py

```diff
@@ -108,5 +107,7 @@ def get_remove_columns_from_dataset(
             remove_columns.remove(self.text_column)
         if self.PROMPT_KEY in remove_columns:
             remove_columns.remove(self.PROMPT_KEY)
+        if "mask" in remove_columns:
```

Comment:
nit: could we make "mask" a constant like we do for self.PROMPT_KEY? It makes it easier to access the column in other parts of the code